src/css/index.css

```diff
@@ -1,19 +1,3 @@
-a:empty,
```

Comment:
In this way, there's a chance that we unintentionally override the style in user's website where the widget is being used. 

So I wrapped this config under a certain namespace; `aichatbot` which is a root component's id. https://github.com/sendbird/chat-ai-widget/blob/develop/packages/self-service/src/main.tsx#L6